### DISCUSSION TITLE
enable/disable pressure-sensitive

### DISCUSSION THREAD

**Discussion by tobias0409** [personalizedrefrigerator/js-draw#65] [Ideas & Feature Requests]
Hello,

the function to let the user decide whether he wants to work pressure-sensitive or not, in the form of a button that protects the drawing window from rotation would be great.

Working on a Samsung Tab S7 FE.

**Comment by personalizedrefrigerator** [comment]
> the function to let the user decide whether he wants to work pressure-sensitive or not

I think it might make sense to add an "advanced settings" screen for this. Here's a possible UI:

![UI mockup: three dots to the right of the color picker allow opening an advanced settings menu](https://github.com/personalizedrefrigerator/js-draw/assets/46334387/8d3f8fc0-94eb-402e-8484-ba6462906311)


> a button that protects the drawing window from rotation would be great.

This feature may already exist under the "pan" tool menu:

![screenshot: Lock rotation option highlighted under the "pan" tool menu.](https://github.com/personalizedrefrigerator/js-draw/assets/46334387/de5df501-5c24-4ccc-a90a-7fed9b468ddb)
